docs/docs/configuration/record.md

```diff
@@ -22,17 +45,11 @@ record:
   enabled: True
   events:
     retain:
-      default: 10
+      default: X
```

Comment:
Why change this to X?

Reply:
Maybe the Roman numeral system is now supported for options. 😆
Indeed, it's better to keep a real value such as "1.5" to cover the decimal support and avoid "where X is a number".

Reply:
My thinking was I didn't want to use the default value for 24/7 recording because some people like to copy configs without reading but also wasn't sure if it would be good to choose a random value, but then I wanted the other part to be consistent